packages/support/src/settings/_variables.color.scss

```diff
@@ -1,145 +1,22 @@
-        <code class="c-swatch__label">
-          $color-<%= color %>
-        </code>
-      </article>
-    <% }) -%>
-  </div>
-
-Style guide: style.color.secondary
-*/
-
-/*
-Background colors
-
-@hide-markup
-
-These colors are used largely for background blocks and large content areas. When alternating between tones, be sure to use enough contrast between adjacent colors.
-
-Markup:
-<% var colors = [
-  'background', 'background-inverse', 'gray-dark', 'gray', 'gray-light', 'gray-lighter', 'gray-lightest'
-]; -%>
-<div class="ds-l-container">
-  <div class="ds-l-row">
-    <% colors.forEach(color => { -%>
-      <article class="ds-l-col--6 ds-l-md-col--4 ds-l-lg-col--3 ds-u-padding-x--1 ds-u-padding-bottom--2 c-swatch">
-        <div class="c-swatch__preview ds-u-fill--<%= color %>"></div>
-        <strong class="js-swatch-hex">#</strong>
-        <code class="c-swatch__label">
-          $color-<%= color %>
-        </code>
-      </article>
-    <% }) -%>
-  </div>
-
-Style guide: style.color.background
-*/
-
-/*
-Status colors
-
-@hide-markup
-
-These colors are used largely to indicate the status of something. Be cautious using these colors to signify something that may contradict what most people would expect the colors to be used for.
-
-Markup:
-<% var colors = [
-  'success', 'success-light', 'success-lighter', 'success-lightest',
-  'warn', 'warn-light', 'warn-lighter', 'warn-lightest',
-  'error-darkest', 'error-dark', 'error', 'error-light', 'error-lighter', 'error-lightest',
-]; -%>
-<div class="ds-l-container">
-  <div class="ds-l-row">
-    <% colors.forEach(color => { -%>
-      <article class="ds-l-col--6 ds-l-md-col--4 ds-l-lg-col--3 ds-u-padding-x--1 ds-u-padding-bottom--2 c-swatch">
-        <div class="c-swatch__preview ds-u-fill--<%= color %>"></div>
-        <strong class="js-swatch-hex">#</strong>
-        <code class="c-swatch__label">
-          $color-<%= color %>
-        </code>
-      </article>
-    <% }) -%>
-  </div>
-
-Style guide: style.color.status
-*/
-
 // Generic
 // USWDS doesn't include a "lighter" variant for red, so we create one
 // in order to be consistent with the other "state" colors.
 $color-secondary-lighter: mix($color-secondary-light, $color-secondary-lightest)
   !default;
 
+// USE COLOR RED HERE
 $color-error: $color-secondary !default;
 $color-error-dark: $color-secondary-dark !default;
 $color-error-darkest: $color-secondary-darkest !default;
 $color-error-light: $color-secondary-light !default;
 $color-error-lighter: $color-secondary-lighter !default;
 $color-error-lightest: $color-secondary-lightest !default;
 
+$color-red: $color-secondary !default;
+$color-red-light: $color-secondary-light !default;
+$color-red-lighter: $color-secondary-lighter !default;
+$color-red-lightest: $color-secondary-lightest !default;
+
 $color-warn: $color-gold !default;
 $color-warn-light: $color-gold-light !default;
 $color-warn-lighter: $color-gold-lighter !default;
```

Comment:
This might be out of scope for this ticket, but I think it makes sense to make sure all status colors have `darker` and `dark` variants. Currently it looks like success and error both do, but warn doesnt. Additionally, only error actually has css variables defined for those variants, while success defines the variants inline in `_variables.color.scss` (as opposed to in `_variables.scss`). What do you think about filing an additional ticket to make all that consistent?

Reply:
https://jira.cms.gov/browse/WNMGDS-331